project/Settings.scala

```diff
@@ -0,0 +1,39 @@
+import sbt._
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+
+/**
+ * Application settings. Configure the build for your application here.
+ * You normally don't have to touch the actual build definition after this.
+ */
+object Settings {
+  /** The name of your application */
+  val name = "anduin-fsm"
```

Comment:
I copied this from another github project 😄 

Could you tell me if I need to add anything to `build.sbt`, as I can't find any variables that hold this string in the current `build.sbt`?